themes/media-theme-netflix.js

```diff
@@ -0,0 +1,164 @@
+import MediaTheme from './media-theme.js';
+
+const template = `
+<style>
+  body {
+    font-family: "Helvetica Neue", sans-serif;
+  }
+
+  media-controller {
+    width: 100%;
+    /* Keep the buttons from overflowing */
+    min-width: 420px;
+    height: 720px;
+
+    --media-range-thumb-background: rgba(255,0,0, 1);
+    --media-range-track-height: 4px;
+    --media-range-track-transition: height .2s ease;
+    --media-range-track-background: #555;
+    --media-range-bar-color: rgb(229, 9, 20);
+
+    --media-button-icon-width: 50px;
+    --media-button-icon-height: 50px;
+  }
+
+  media-time-range {
+    width: 100%;
+    height: 25px;
+    margin-bottom: 10px;
+    --media-range-thumb-height: 20px;
+    --media-range-thumb-width: 20px;
+    --media-range-thumb-border-radius: 20px;
+    --media-time-buffered-color: #777;
+  }
+
+  media-time-range:hover {
+    --media-range-track-height: 9px;
+  }
+
+  media-control-bar {
+    background: none;
+    justify-content: space-between;
+    -webkit-box-align: center;
+    -ms-align-items: center;
+    align-items: center;
+    display: flex;
+    -webkit-box-pack: justify;
+    flex-wrap: nowrap;
+  }
+
+  media-control-bar > * {
+    background: none;
+    display: flex;
+    flex: 0 1 auto;
+    width: 60px;
+    min-width: 60px;
+    height: 80px;
+    padding-bottom: 20px;
+    margin: 0 3px;
+
+    --media-button-icon-transform: scale(1.2);
+    --media-button-icon-transition: transform .2s ease;
+  }
+
+  /* For some reason media-control-bar > *:hover doesn't work...
+  while media-control-bar > *:focus-within does. 
+  And also media-control-bar > *:not(:hover)
+  Really annoying. Need to submit a bug.  */
+  media-control-bar > *:not(:hover) {
+    --media-button-icon-transform: scale(1);
+    --media-button-icon-transition: transform .2s ease;
+  }
+
+  media-play-button,
+  media-seek-backward-button,
+  media-seek-forward-button,
+  media-mute-button,
+  media-fullscreen-button {
+    height: 80px;
+  }
+
+  media-fullscreen-button {
+    margin-right: 10px;
+  }
+  
+  media-control-bar > *:focus,
+  media-control-bar > *:focus-within {
+    outline: 0;
+  }
+
+  media-volume-range {
+    width: 100px;
+  }
+
+  .videoTitle {
+    flex-grow: 1;
+    height: 80px;
+    line-height: 64px;
+    vertical-align: middle;
+    overflow: hidden;
+    padding: 0 10px;
+    min-width: 0;
+  }
+
+  .videoTitleText {
+    width: 100%;
+    vertical-align: middle;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+  }
+
+  .videoTitleText h4 {
+    margin: 0;
+    display: inline-block;
+    white-space: nowrap;
+  }
+
+  .videoTitle span {
+    display: inline;
+    font-weight: 200;
+    margin-left: 5px;
+  }
+
+</style>
+
+<media-controller>
+
+  <slot name="media" slot="media"></slot>
```

Comment:
This is the thing that feels nuckin futs. Chaining slots. But it works!

Reply:
Whoa.

So from the outside, an application does this:

```html
<media-theme-netflix>
  <video slot="media"></video>
</media-theme-netflix>
```

That "media" slot gets passed through into the "media" slot for `media-controller`. Pretty nuckin futs